pkg/services/queryhistory/models.go

```diff
@@ -98,3 +87,26 @@ type QueryHistoryMigrationResponse struct {
 	TotalCount   int    `json:"totalCount"`
 	StarredCount int    `json:"starredCount"`
 }
+
+// CreateQueryInQueryHistoryCommand is the command for adding query history
+// swagger:model
+type CreateQueryInQueryHistoryCommand struct {
+	// UID of the data source for which are queries stored.
+	DatasourceUID string `json:"datasourceUid"`
+	// The JSON model of queries.
+	Queries *simplejson.Json `json:"queries"`
+}
```

Comment:
Optionally, you can provide an examples values for the fields:
```suggestion
// CreateQueryInQueryHistoryCommand is the command for adding query history
// swagger:model
type CreateQueryInQueryHistoryCommand struct {
	// UID of the data source for which are queries stored.
	// example: PE1C5CBDA0504A6A3
	DatasourceUID string `json:"datasourceUid"`
	// The JSON model of queries.
	// required: true
	// example: [ { "datasourceUid": "PE1C5CBDA0504A6A3", "queries": [ { "refId": "A", "key": "Q-87fed8e3-62ba-4eb2-8d2a-4129979bb4de-0", "scenarioId": "csv_content", "datasource": { "type": "testdata", "uid": "PD8C576611E62080A" } } ], "starred": false, "createdAt": 1643630762, "comment": "debugging" } ] 
	Queries Data `json:"queries"`
}

// swagger:model
type Data *simplejson.Json
```
and then in the swagger editor will appear like:
<img width="1234" alt="Screenshot 2022-06-09 at 2 17 40 PM" src="https://user-images.githubusercontent.com/1632407/172834974-d6a05c13-01f6-4024-b79e-98251616bbce.png">